docs/release/trg-9/trg-9-01.md

```diff
@@ -0,0 +1,76 @@
+---
+title: TRG 9.01 - UX consistency / Styleguide for UI
+sidebar_position: 1
+---
+
+| Status | Created       | Post-History                                             |
+|--------|---------------|----------------------------------------------------------|
+| Active | 22-Jul-2024   | Finalize TRG                                             |
+| Draft  | 25-April-2024 | Added new styleguide criteria - initial version          |
+| Draft  | 10-Jan-2024   | Transferred from Release Acceptance Criteria into TRG    |
+
+## Introduction
+
+The purpose of this guideline is to ensure that business applications with user interfaces adhere to the established UI/UX styleguide which is collected in the [Portal Shared Components](https://github.com/eclipse-tractusx/portal-shared-components.git). It outlines all the relevant criteria that must be checked when releasing a new product version. A crosscheck with every pull request is not just optional, but it is a "MUST" criteria to ensure the quality and consistency of the user interfaces of each product/component. This step is crucial in maintaining the overall standard and enhancing as well as standardizing the user experience.
+
+## Objective
+
+The objective is to ensure consistency, usability, and adherence to design standards across all user interfaces within the business application.
+
+## Criteria (Basic)
+
+### Styleguide Compliance
+
+- All user interface elements must match the predefined styles. This includes fonts, colors, button styles, input fields, and layout grids.
+  - [Storybook](https://eclipse-tractusx.github.io/portal-shared-components)
```

Comment:
I like the idea that there is a really clear style guide, however not the same components (from react.js) should be used. Maybe when you are using vue, you can use the native components and just adapt the style to look like in the "style guide". Maybe to mention this could help.